src/test/java/seedu/address/logic/commands/FindCommandTest.java

```diff
@@ -22,8 +22,8 @@
  * Contains integration tests (interaction with the Model) for {@code FindCommand}.
  */
 public class FindCommandTest {
-    private Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs(), new UniCash());
-    private Model expectedModel = new ModelManager(getTypicalAddressBook(), new UserPrefs(), new UniCash());
+    private final Model model = new ModelManager(new UniCash(), new UserPrefs());
+    private final Model expectedModel = new ModelManager(new UniCash(), new UserPrefs());
     @Test
     public void equals() {
         TransactionNameContainsKeywordsPredicate firstPredicate =
```

Comment:
Now that we've deleted the original NameContainsKeywordsPredicate, do we want to change the unsightly TransactionNameContainsKeywordsPredicate back or keep it the same for now?